kshebeko/lesson4/src/com/company/kshebeko_class/Bouquet.java

```diff
@@ -0,0 +1,47 @@
+package com.company.kshebeko_class;
+
+import java.util.ArrayList;
+
+public class Bouquet {
+    public Accessories accessories;
+    public String freshFlower;
+    public int price;
+    public String name;
+
+    public Bouquet (String name, int price){ //Конструктор
+        this.name = name;
+        this.price = price;
+        System.out.println(this.name + this.price);
+    }
+
+    public Bouquet(String freshFlower) { //Конструктор
+        this.freshFlower = freshFlower;
+    }
+
+    public ArrayList<Flower> flowerList = new ArrayList();
+
+    public void setFlowerList(ArrayList<Flower> list) {
+        this.flowerList = list;
+    }
+
+    public Bouquet() {
```

Comment:
if there are several methods of same type(like constructors in this case) they should go in some sort of group next to each other. When they are separated by smth else it makes code less readable.
Same for fields - they are  usually placed at the very top of a class (like all fields in this class except flowerList )